pkg/storaged/stratis-details.jsx

```diff
@@ -741,6 +974,26 @@ export const StratisStoppedPoolDetails = ({ client, uuid }) => {
                         <DescriptionListTerm>{_("storage", "UUID")}</DescriptionListTerm>
                         <DescriptionListDescription>{ uuid }</DescriptionListDescription>
                     </DescriptionListGroup>
+                    { encrypted &&
+                    <DescriptionListGroup>
+                        <DescriptionListTerm className="control-DescriptionListTerm">
+                            {_("storage", "Passphrase")}
+                        </DescriptionListTerm>
+                        <DescriptionListDescription>
+                            { keydesc ? cockpit.format(_("using key description $0"), keydesc) : <i>{_("not configured")}</i> }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230601-113309-0e7aeb29-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)